sample-client-app/sample-app-with-shared-lib/build-spring-boot.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+plugins {
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+	id 'java'
+}
+
+group = 'com.vmware.vipclient'
```

Comment:
Since you revert the change for group, so this group should be consistent with others.